packages/signers/src/index.ts

```diff
@@ -1,7 +1,10 @@
+export * from './account-signer-meta';
+export * from './deduplicate-signers';
```

Comment:
y exported?

Reply:
Cause I documented it haha. I'll remove it from the `index.ts`. 👍